BitwardenShared/Core/Vault/Repositories/VaultRepository.swift

```diff
@@ -1120,6 +1161,17 @@ extension DefaultVaultRepository: VaultRepository {
         return organizations.first(where: { $0.id == id })
     }
 
+    func getItemTypesUserCanCreate() async -> [CipherType] {
+        let itemTypes: [CipherType] = CipherType.canCreateCases.reversed()
```

Comment:
🤔 I'm not 100% sure whether we should be using `reversed` here or not... On one hand the order of the array is based on which UI that's presented therefore it should be done by the processor or state of such view. But on the other hand we're already ordering other types like ciphers, folders and others in the `VaultRepository` for the vault list section items. I guess for the time being this is fine given it centralizes the operation on the current callers.
What do you think? @KatherineInCode @matt-livefront